src/components/details.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import WorldMap from '../assets/worldmap.png';
+
+const Detail = () => {
```

Comment:
Kindly take a look at the image below
- [ ] As is highlighted in the first image, unlike the second image, some of your countries' renderings are missing the additional information. Kindly fix this.
- [ ] In the second image, the seems to be an issue with the responsiveness of your details page after a certain screen width. It would be nice to fix this. Using CSS grid or flexbox can help with this.

![amal-details](https://user-images.githubusercontent.com/36542986/142729620-ce84dad1-6c15-4196-8415-db6a6d9620c5.jpg)